packages/core/test/RoutingNode.test.ts

```diff
@@ -1901,7 +1949,7 @@ describe('RoutingNode', () => {
 					nodes: workflowData.nodes,
 					connections: workflowData.connections,
 					active: false,
-					nodeTypes,
+					nodeTypes: mock(),
```

Comment:
Why do we pass a mock here if we have an instance available? 
I removed the mock and the test still passes.